pkg/config/auth.go

```diff
@@ -285,6 +291,265 @@ func (s *sms) fromAuthConfig(remoteConfig v1API.AuthConfigResponse) {
 	}
 }
 
+func (e external) toAuthConfigBody(body *v1API.UpdateAuthConfigBody) {
+	if len(e) == 0 {
+		return
+	}
+	var p *provider
+	// Ignore configs of disabled providers because their envs are not loaded
```

Comment:
**remark**

I think that might be a marker that we wanna refactor the way we load the env/config 🤔 

One approach would be to always load it everywhere it's mentioned, whether the actual service is enabled or not. And if it raise an error, then the user can just "comment" or remove the errored line if he still want to keep the thing around.

Reply:
Yea possibly. We would have to consider this change carefully though because it could break CI/CD pipelines that rely on setting `enabled = false` but not commenting out the usage of env.